zgrab_schema.py

```diff
@@ -205,7 +205,11 @@
         "value":Binary(),
         "length":Integer(),
         "lifetime_hint":Long()
-    })
+    }),
+    "crypto_variables":SubRecord({
```

Comment:
There is no current client_key_exchange JSON output record for TLS. The only related output record is the server_key_exchange which contains the server's public values and signature. I can create a new client_key_exchange JSON record including the client's public and private values if needed. Putting the premaster secret there kinda makes sense, but not completely b/c it is dependent on both the server KEX and client KEX records' values.

The same goes with the master secret. It kinda makes sense to put it in the server_key_exchange record, but not completely b/c it's not wholly based on the server KEX. I could put it in the server_finished b/c that's the point in the handshake that it's required, but it doesn't make logical sense.

That's why I created a new data->tls layer record to hold these values. They're dependent on multiple parts of the existing JSON record structure and don't fit well WRT picking one to insert it into or the common-sense lay out of the existing output.